lib/manager/jenkins/extract.ts

```diff
@@ -1,12 +1,69 @@
+import is from '@sindresorhus/is';
+import yaml from 'js-yaml';
 import * as datasourceJenkins from '../../datasource/jenkins-plugins';
 import { logger } from '../../logger';
 import { SkipReason } from '../../types';
 import { isSkipComment } from '../../util/ignore';
 import * as dockerVersioning from '../../versioning/docker';
 import type { PackageDependency, PackageFile } from '../types';
 
-export function extractPackageFile(content: string): PackageFile | null {
-  logger.trace('jenkins.extractPackageFile()');
+function getDependency(
+  name: string,
+  version: string,
+  url: string,
+  groupId: string
+): PackageDependency {
+  const dep: PackageDependency = {
+    datasource: datasourceJenkins.id,
+    versioning: dockerVersioning.id,
+    depName: name,
+  };
+
+  if (version) {
+    dep.currentValue = version.toString();
+  } else {
+    dep.skipReason = SkipReason.NoVersion;
+  }
+
+  if (version === 'latest' || version === 'experimental' || groupId) {
```

Comment:
Why  unsupported with `groupId`

Reply:
Because when groupId is specified, we should use the incremental jenkins registry (datasource?) instead of the official one. At the moment, only the default official one is used. I dont have much knowledge on renovate, so I think this could be a feature in the future, but is not required for this PR.